requirements.txt

```
@@ -1,7 +1,7 @@
 aiohttp>=3.9,<4.0
 Pillow
 pystray
-PyGObject<3.47; sys_platform == "linux" # required for better system tray support on Linux
+PyGObject; sys_platform == "linux" # required for better system tray support on Linux
```

This is probably not the right place to doctorize about it, but a standard Pylint in-line comment consists of two spaces, followed by a hash, followed by a space, and then the actual comment. This isn't Python code, but the rest of the project follows this standard, so I'd like to keep it here as well. Since this line is getting touched already, it might as well be made "the right way" now.
```suggestion
PyGObject; sys_platform == "linux"  # required for better system tray support on Linux
```